ui/src/timeMachine/components/variableToolbar/VariableTooltipContents.tsx

```diff
@@ -105,7 +106,21 @@ const VariableTooltipContents: FunctionComponent<Props> = ({
 
 const mstp = (state: AppState, ownProps: OwnProps) => {
   const valuesStatus = getTimeMachineValuesStatus(state)
-  const values = getTimeMachineValues(state, ownProps.variableID)
+  const {variableID} = ownProps
+  let values: VariableValues
+  if (variableID.includes('timeRange') || variableID.includes('windowPeriod')) {
```

Comment:
This code is a perfect example of something that should live inside a selector and unit tested.  Shoot, it could even go inside the `getTimeMachineValues` selector.  Generally, anytime I'm doing a whole bunch of logic inside of `mstp` I like to ask myself "should this go into a selector?"

Reply:
Sounds good to me